### PR TITLE
reduce model zoo test size

### LINKED ISSUE #7293: test_gluon_rnn.test_rnn_cells hang
For bugs or installation issues, please provide the following information.
The more information you provide, the more likely people will be able to help you.

## Environment info
Operating System: DL AMI

Compiler:

Package used (Python/R/Scala/Julia):

MXNet version:

Or if installed from source:

MXNet commit hash (`git rev-parse HEAD`):  861e929cec8fa8fbab06884d9605debb74cd7217

If you are using python package, please provide

Python version and distribution:

If you are using R package, please provide

R `sessionInfo()`:

## Error Message:
Please paste the full error message, including stack trace.
```
.... 
test_executor.test_dot ... ok
test_executor.test_reshape ... ok
test_gluon_data.test_array_dataset ... ok
test_gluon_data.test_recordimage_dataset ... --2017-08-01 18:07:26--  http://data.mxnet.io/data/test_images.tar.gz
Resolving data.mxnet.io (data.mxnet.io)... 54.208.175.7
Connecting to data.mxnet.io (data.mxnet.io)|54.208.175.7|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 2211939 (2.1M) [application/x-gzip]
Saving to: ‘data/test_images.tar.gz’

100%[=================================================>] 2,211,939   2.57MB/s   in 0.8s

2017-08-01 18:07:27 (2.57 MB/s) - ‘data/test_images.tar.gz’ saved [2211939/2211939]

ok
test_gluon_data.test_sampler ... ok
test_gluon_model_zoo.test_concurrent ... ok
test_gluon_model_zoo.test_identity ... ok
test_gluon_model_zoo.test_models ... ok
test_gluon_rnn.test_rnn ... ok
test_gluon_rnn.test_lstm ... ok
test_gluon_rnn.test_lstm_forget_bias ... ok
test_gluon_rnn.test_gru ... ok
test_gluon_rnn.test_residual ... ok
test_gluon_rnn.test_residual_bidirectional ... ok
test_gluon_rnn.test_stack ... ok
test_gluon_rnn.test_bidirectional ... ok
test_gluon_rnn.test_zoneout ... ok
test_gluon_rnn.test_rnn_cells ... 
```
## Minimum reproducible example
if you are using your own code, please provide a short script that reproduces the error.

## Steps to reproduce
or if you are running standard examples, please provide the commands you have run that lead to the error.

1. run `nosetests-2.7 -v ./tests/python/unittest/`, it hangs when it comes to `test_gluon_rnn.test_rnn_cells`. 
2.
3.

## What have you tried to solve it?

1. run `nosetests-2.7 -v ./tests/python/unittest/test_gluon_rnn.py`, but it successfully passes the test
2.
3.

### PR DESCRIPTION
**Pull request by szha**
this should fix #7293

### CODE REVIEW

File: tests/python/unittest/test_gluon_model_zoo.py [hunk 1]
```diff
@@ -43,11 +46,11 @@ def test_models():
     for model_name in all_models:
         test_pretrain = model_name in pretrained_to_test
         model = get_model(model_name, pretrained=test_pretrain)
-        data_shape = (7, 3, 224, 224) if 'inception' not in model_name else (7, 3, 299, 299)
-        print(model)
+        data_shape = (2, 3, 224, 224) if 'inception' not in model_name else (2, 3, 299, 299)
+        eprint('testing forward for %s'%model_name)
```
Comment (eric-haibin-lin) [inline tests/python/unittest/test_gluon_model_zoo.py]: is the print still necessary?

Comment (szha) [inline tests/python/unittest/test_gluon_model_zoo.py]: this helps people see that the tests are still running and didn't hang.

**Comment by piiswrong** [comment]
what's eprint

**Comment by szha** [comment]
print to stderr.

**Pull request merged**